scripts/exercice-1.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
In fact, iy might even easier to just remove the shebang and the utf8 coding as well (for all scripts).
Could you do it ?